test/Microsoft.ML.Functional.Tests/ONNX.cs

```diff
@@ -109,7 +104,7 @@ public void SaveOnnxModelLoadAndScoreKMeans()
             // TODO #2980: ONNX outputs don't match the outputs of the model, so we must hand-correct this for now.
             // TODO #2981: ONNX models cannot be fit as part of a pipeline, so we must use a workaround like this.
             var onnxWorkaroundPipeline = onnxModel.Append(
-                mlContext.Transforms.CopyColumns("Score", "Score.onnx").Fit(onnxModel.Transform(data)));
+                mlContext.Transforms.CopyColumns("Score", "Score").Fit(onnxModel.Transform(data)));
```

Comment:
>CopyColumns("Score", "Score") [](start = 37, length = 29)

Do we need CopyColumn now?

Reply:
We don't need CopyColumns either. 

---
In reply to: [381300171](https://github.com/dotnet/machinelearning/pull/4849#discussion_r381300171) [](ancestors = 381300171)